### ISSUE TITLE
Total climb data should sum all the climbs

### ISSUE THREAD

**Issue by epan** [cogsci/wigglemap#77]
Currently it seems to show the net climb. Does it make more sense to show the total amount of climbing to be done rather than the net climb minus the downhill? We got feedback wanting to see the sum climb and that's how Strava and other apps work, I believe.

@andrewdanks @slinlee @dhchow

**Comment by andrewdanks** [comment]
Yeah, we can do this (kind of). If we continue to use Google Elevations API, then there's a limit to how accurate it can be, because all we can get are elevations at start and end points of some segment in the route, getting the total climb out of this assumes that there was no downhill between the two points.

Is there a more accurate way of calculating this?

**Comment by andrewdanks** [comment]
And this isn't a bug, per se, but an enhancement.

**Comment by slinlee** [comment]
For now it sounds fine to use the elevation from the start/end points of a
segment. Is it possible to split  segments more by asking for things like,
"from here to 100 ft towards the end of the segment" ?

**Comment by andrewdanks** [comment]
Hmm, not that I can think of. I don't know how we'd get the longitude/latitude coordinates of a point X ft away from another coordinate. Maybe it's a little possible if we also have the heading, but then we assume there are no turns in between. Sounds complicated, lol, or maybe I'm over thinking this.

**Comment by epan** [comment]
How did the WSJ do it? I linked the blog post.

(mobile)

**Comment by kmiyashiro** [comment]
https://developers.google.com/maps/documentation/javascript/elevation
